metrics_collector/volume_collector.go

```diff
@@ -251,6 +265,14 @@ func getVolumeRobustnessValue(v *longhorn.Volume) int {
 	return robustnessValue
 }
 
+func getLastBackupAtValue(v *longhorn.Volume) int64 {
+	lastBackupAt, err := time.Parse(time.DateTime, v.Status.LastBackupAt)
```

Comment:
Doesn't look like it will parse successfully, probably need to use a different layout.